src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,7 @@
+rootLogger.level = info
```

Comment:
I don't think the main jar should come with a `log4j2.properties` file. The "application" that uses / executes this jar should provide the logging configuration. So the test jar and e2e test "execute" this jar, they are fine to define their logging.

Reply:
makes sense, removed